server/app/views/admin/programs/ManageProgramAdminsView.java

```diff
@@ -64,72 +71,105 @@ public Content render(
             .setTitle(fullTitle)
             .addMainContent(
                 renderHeader(fullTitle),
-                adminEmailTemplate(),
-                renderAdminForm(request, program.id(), existingAdminEmails));
+                renderHeader("Add New Admin"),
+                renderAddNewButton(request, program.id()),
+                renderHeader("Existing Admins"),
```

Comment:
renderHeader adds an h1 tag and there should only be one h1 tag per page. Instead, you probably want to use h2() for "Add New Admin" and "Existing Admins"

Reply:
Done